CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 0.5.0-dev
   **NEW FEATURES**
   - [#94](https://github.com/Datatamer/unify-client-python/issues/94) Add access to Attributes of a Dataset
+  - [#103](https://github.com/Datatamer/unify-client-python/issues/103) Dataset `update_records` improvements
```

Comment:
Can you add more detail here? Perhaps something like "`update_records` now returns the JSON response body for the underlying `POST datasets/{id}:updateRecords` call"